src/editor/EditorCommandHandlers.js

```diff
@@ -207,7 +209,9 @@ define(function (require, exports, module) {
         if (containsNotLineComment) {
             // Comment out - prepend the first prefix to each line
             for (i = startLine; i <= endLine; i++) {
-                editGroup.push({text: prefixes[0], start: {line: i, ch: 0}});
+                // For each line we need to determine the position where de comment prefix should be inserted
+                prefixPos = _getPrefixPos(language, doc.getLine(i));
+                editGroup.push({text: prefixes[0], start: {line: i, ch: prefixPos}});
             }
 
             // Make sure tracked selections include the prefix that was added at start of range
```

Comment:
It looks like this code that deals with `trackedSel` contains assumptions about the comment being inserted at `ch: 0`.  You may need to update it to ensure it works as intended in the new case too.